src/controllers/v1/profile.js

```diff
@@ -162,6 +162,33 @@ module.exports = class Mentees {
 			return error
 		}
 	}
+	/**
+	 * Fetches user profile details.
+	 * @method
+	 * @name details
+	 * @param {Object} req - Request object.
+	 * @param {Object} req.params - Route parameters.
+	 * @param {String} req.params.id - The mentor's ID.
+	 * @param {Object} req.decodedToken - Decoded token from authentication.
+	 * @param {String} req.decodedToken.id - The user's ID.
+	 * @param {String} req.decodedToken.organization_id - The user's organization ID.
+	 * @param {Array} req.decodedToken.roles - The user's roles.
+	 * @param {Boolean} isAMentor - Indicates whether the user is a mentor.
+	 * @returns {Promise<Object>} - The mentor's profile details.
+	 */
+	async details(req) {
```

Comment:
@nevil-mathew why this API needed ?

Reply:
Currently, we only have API to get mentor details, this API allows getting any user details. Will update the JS Doc to mention that.